tests/opentracer/test_tracer_asyncio.py

```diff
@@ -8,37 +8,34 @@
 from ddtrace.opentracer.utils import get_context_provider_for_scope_manager
 
 from tests.contrib.asyncio.utils import AsyncioTestCase, mark_asyncio
-from .conftest import dd_tracer, ot_tracer_factory, writer
+from .conftest import ot_tracer_factory
 
 
 @pytest.fixture()
-def ot_tracer(ot_tracer_factory):
-    return ot_tracer_factory(
+def ot_tracer(request, ot_tracer_factory):
+    # use the dummy asyncio ot tracer
+    request.instance.ot_tracer = ot_tracer_factory(
```

Comment:
The `BaseTracerTestCase` we are using here defines `self.tracer` as a "dummy tracer". The fixture here is overriding that. Should we instead to assume the same interface for this fixture, i.e. `request.instance.trace = ot_tracer_factor(...)` and below call `self.tracer` and `self.tracer.writer` as we do with `BaseTracerTestCase`?

Reply:
Yes… but no. I actually tried to do that, but the `setUp` method is called *after* the fixtures. Which means the fixtures is overridden.  😿